c/spot/source/GPOS.c

```diff
@@ -3565,16 +3574,16 @@ static void showContext2(ContextPosFormat2 *fmt, IntX level, void *feattag)
 							char name2[MAX_NAME_LEN];
 							GlyphId glyphId2 = *da_INDEX(CR->glyphidlist, c);
 							strcpy(name2, getGlyphName(glyphId2, forProofing));
-                              if ((psIndex + strlen(name2) + 10) >= kProofBufferLen)
-                              {
-                                  if (level == 7)
-                                      fprintf(OUTPUTBUFF, "\n    %s", proofString);
-                                  else if (level == 8)
-                                      dumpMessage(proofString, ((Card32 *)feattag)[0]);
-                                  proofString[0] = 0;
-                                  psIndex = 0;
-                              }
-							psIndex = strlen(proofString);
+							  psIndex = strlen(proofString);
+							  if ((psIndex + strlen(name2) + 10) >= kProofBufferLen)
+							  {
+								  if (level == 7)
+									  fprintf(OUTPUTBUFF, "\n	 %s", proofString);
```

Comment:
Please change line 3581 to use spaces instead of a tab character in the output string.
(Line 3592, which is otherwise identical, uses spaces instead of a tab in the output string.)